spec/support/evm_spec_helper.rb

```diff
@@ -44,7 +44,6 @@ def self.clear_caches
     clear_instance_variable(Tenant, :@root_tenant) if defined?(Tenant)
 
     EmsEvent.clear_event_groups_cache if defined?(EmsEvent)
-    EventStream.clear_event_groups_cache if defined?(EventStream)
     MiqEvent.clear_event_groups_cache if defined?(MiqEvent)
```

Comment:
Alternateively, if we implement the case class as clearing all the descendant classes, then this can be:

```suggestion
    EmsEvent.clear_event_groups_cache if defined?(EmsEvent)
```

Reply:
yeah, I like it